src/serde_ron.rs

```diff
@@ -147,13 +153,14 @@ impl std::fmt::Debug for DeRonErr {
     }
 }
 
-impl std::fmt::Display for DeRonErr {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        std::fmt::Debug::fmt(self, f)
+impl core::fmt::Display for DeRonErr {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        core::fmt::Debug::fmt(self, f)
     }
 }
 
-impl std::error::Error for DeRonErr {}
+// NOTE: core::error::Error requires nightly for now
```

Comment:
I am totally OK to wait a bit and wait just after the Error is going to be stabilized. 

However, maybe it is possible to put it under the same cargo feature-gate as the `hashbrown`? This way current users with outdated compilers will not really notice any difference, while it will be possible to enable this feature and get no_std support. (I am not sure if this is a good idea)

Reply:
Yeah, it's totally fine to put it under `no_std` feature gate. Backwards compatibility is a good thing.

However, in order to use `std::error::Error` and `std::collections::HashMap`, macro `#![no_std]` have to be removed. If I get it correctly, besides from blocking any use of std, it doesn't link it when used in no_std program. Not sure if crate level directives can be future gated because never tried that personally.

Reply:
Looks `no_std` could be feature-gated: `#![cfg_attr(not(feature = "use_std"), no_std)]` instead of `![no_std]` should do it